### ISSUE TITLE
Compute recovered loss from the bridge identity to avoid cancellation

### ISSUE THREAD

**Issue by zuizui0223** [zuizui0223/payoff#136]
## Reproduced cancellation in the core recovery estimand

`QuadraticTraitArchitecture.recovered_loss` currently returns

```python
conflict_load - differentiated_loss
```

while the same class already exposes the exact bridge identity

```text
recovered_loss = separation_fraction * conflict_load.
```

At strong coupling the differentiated loss approaches the conflict load, so the subtraction can lose the entire small recovered signal even at otherwise ordinary parameter scales.

Concrete reproduction:

```text
a=b=1
theta1-theta2=1
coupling=1e16

conflict_load        = 0.5
differentiated_loss  = 0.5   # rounded
subtraction recovery = 0.0
bridge recovery      = 2.5e-17
```

The recovered loss is a primary PAYOFF estimand feeding `phi`; collapsing it to zero can therefore alter criticality when architecture costs are comparably small.

## Repair contract

- compute `recovered_loss` directly as `separation_fraction * conflict_load`;
- keep `differentiated_loss` as its own estimand and retain an audit identity test where subtraction is numerically resolvable;
- add strong-coupling regressions showing positive recovery survives beyond the subtraction-cancellation regime;
- require `phi` and critical comparisons to use the stable recovered-loss value;
- preserve all ordinary-scale results within numerical precision.

No theorem or biological model changes.
